src/stripe-payment.js

```diff
@@ -0,0 +1,220 @@
+/**
+ * `stripe-payment`
+ * `Payment broker for stripe`
+ *
+ * @demo demo/index.html
+ * @customElement `stripe-payment`
+ */
+import { LitElement, html, css } from "lit-element/lit-element.js";
+import { ifDefined } from "lit-html/directives/if-defined.js";
+
+class StripePayment extends LitElement {
+  /**
+   * Convention
+   */
+  static get tag() {
+    return "stripe-payment";
+  }
+  /**
+   * LitElement render styles
+   */
+  static get styles() {
+    return [
+      css`
```

Comment:
let's import this from a `.css` file, like the other elements do

Reply:
which other elements? example and I'll do it but the `SripeElements.js` seems to just have a style block in a const var

Reply:
nm, made this change too in latest commit